### ISSUE TITLE
Installation not updated for Docbook 5

### ISSUE THREAD

**Issue by putztzu** [openSUSE/daps#502]
### Problem description
https://github.com/openSUSE/daps/blob/develop/INSTALL.adoc

Installation describes installing Docbook 4
Docbook 5 might be significantly different, no longer DTD based
So, can install supporting Docbook 5 work by simply substituting Docbook 5schemas?
Perhaps a specific package list should be provided (which is not now)

This Github issue tracker also suggests 4 related open issues, #253, #246, #294, #285.
The openSUSE 15.1 documentation says that Docbook 5 is being used to create its documentation.

**Comment by fsundermeyer** [comment]
There is no need for special installation instructions - DAPS supports both, DB4 and DB5. You can install both in parallel or only one of them.

DAPS looks at the main file specified in the DC-file (or with -m on the command line) to determine which DB-Version is used.

Locations for the DB4 and DB5 upstream stylesheets as well as to the DB5 upstream schema are configured in /etc/daps/config. They should work out of the box, since they get resolved to file names by using xmlcatalog.
